resolving/npm-resolver/src/index.ts

```diff
@@ -151,6 +157,12 @@ async function resolveNpm (
       return resolvedFromWorkspace
     }
   }
+
+  const resolvedFromJsr = await tryResolveFromJsr(ctx, wantedDependency, opts, defaultTag)
+  if (resolvedFromJsr != null) {
+    return resolvedFromJsr
+  }
+
   const workspacePackages = opts.alwaysTryWorkspacePackages !== false ? opts.workspacePackages : undefined
   const spec = wantedDependency.pref
     ? parsePref(wantedDependency.pref, wantedDependency.alias, defaultTag, opts.registry)
```

Comment:
jsr packages will be less frequent than npm packages, so we should move resolvedFromJsr to below resolve from npm.

Reply:
Like [this](https://github.com/pnpm/pnpm/pull/9358/commits/c46f5809f9400389e70c725976ddc4cd27365003)?

Reply:
yes